coalib/bearlib/languages/definitions/SCSS.py

```diff
@@ -0,0 +1,11 @@
+from coalib.bearlib.languages.Language import Language
+
+
```

Comment:
Blank space not needed.

Reply:
The gap is there in all the files. So i put this gap

Reply:
Since it's not needed, you will have to remove it.